plugins/woocommerce/includes/admin/class-wc-bis-admin.php

```diff
@@ -0,0 +1,513 @@
+
+		// Admin AJAX.
+		require_once WC_ABSPATH . 'includes/admin/class-wc-bis-admin-ajax.php';
+	}
+
+	/**
+	 * Register own version of select2 library.
+	 *
+	 * @return void
+	 */
+	public static function maybe_register_selectsw() {
+		wc_deprecated_function( __METHOD__, '9.9.0' );
+	}
+
+	/**
+	 * Load own version of select2 library.
+	 *
+	 * @return void
+	 */
+	public static function maybe_load_selectsw() {
+		wc_deprecated_function( __METHOD__, '9.9.0' );
+	}
+
+	/**
+	 * Display notice when selectSW library is unsupported.
+	 *
+	 * @return void
+	 */
+	public static function maybe_display_selectsw_notice() {
+		wc_deprecated_function( __METHOD__, '9.9.0' );
+	}
+
+	/**
+	 * Add a message in the WP Privacy Policy Guide page.
+	 *
+	 * @return void
+	 */
+	public static function add_privacy_policy_guide_content() {
+		if ( function_exists( 'wp_add_privacy_policy_content' ) ) {
+			wp_add_privacy_policy_content( 'WooCommerce Back In Stock Notifications', self::get_privacy_policy_guide_message() );
+		}
+	}
+
+	/**
+	 * Message to add in the WP Privacy Policy Guide page.
+	 *
+	 * @return string
+	 */
+	protected static function get_privacy_policy_guide_message() {
+
+		$content = '
+			<div class="wp-suggested-text">' .
+				'<p class="privacy-policy-tutorial">' .
+					__( 'WooCommerce Back In Stock Notifications stores the following information when customers sign up to receive back-in-stock notifications:', 'woocommerce' ) .
+				'</p>' .
+				'<ul class="privacy-policy-tutorial">' .
+					'<li>' . __( 'Customer e-mail.', 'woocommerce' ) . '</li>' .
+					'<li>' . __( 'Sign-up date.', 'woocommerce' ) . '</li>' .
+					'<li>' . __( 'Notification date.', 'woocommerce' ) . '</li>' .
+				'</ul>' .
+				'<p class="privacy-policy-tutorial">' .
+					__( 'This information can be used to personally identify customers, and is stored in the database indefinitely.', 'woocommerce' ) .
+				'</p>' .
+			'</div>';
+
+		return $content;
+	}
+
+	/**
+	 * Add 'Stock Notifications' tab to WooCommerce Settings tabs.
+	 *
+	 * @param  array $settings
+	 * @return array $settings
+	 */
+	public static function add_settings_page( $settings ) {
+
+		$settings[] = include 'settings/class-wc-bis-admin-settings.php';
+
+		return $settings;
+	}
+
+	/**
+	 * Admin scripts.
+	 *
+	 * @return void
+	 */
+	public static function admin_resources() {
+
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+		$version   = Constants::get_constant( 'WC_VERSION' );
+
+		wp_register_style( 'wc-bis-admin', WC()->plugin_url() . '/assets/css/bis-admin.css', array( 'sw-admin-css-select' ), $version );
+		wp_style_add_data( 'wc-bis-admin', 'rtl', 'replace' );
+
+		wp_register_script( 'wc-bis-writepanel', WC()->plugin_url() . '/assets/js/admin/wc-bis-admin' . $suffix . '.js', array( 'jquery', 'jquery-ui-datepicker', 'wp-util', 'sw-admin-select-init', 'wc-backbone-modal' ), $version );
```

Comment:
I'm seeing some errors in Query Monitor related to missing dependencies:

<img width="1059" alt="image" src="https://github.com/user-attachments/assets/872524fc-e554-4a31-9630-ae250c7c7b84" />

<img width="1130" alt="image" src="https://github.com/user-attachments/assets/94bed5ca-d347-427e-bdf1-efdd2c3e2fb4" />

Reply:
Well spotted! I fixed this yesterday but forgot to push. Fixed in [067fdda](https://github.com/woocommerce/woocommerce/pull/55787/commits/067fdda85649fc099a9cd50e6b6392842c6a8537).